ci/deps.sh

```diff
@@ -34,40 +35,72 @@ echo "MAKEFLAGS: ${MAKEFLAGS}"
 mkdir -p /tmp
 cd /tmp
 
+gh_retrieve_wget(){
+    ORG=$1
+    REPO=$2
+    TAG=$3
+
+    echo "Downloading $ORG $REPO $TAG"
+    
+
+
+    wget $WGET_FLAGS https://github.com/$ORG/$REPO/archive/refs/tags/$TAG.tar.gz && \
+    mkdir $REPO && \
+    tar xzfv $TAG.tar.gz --strip-components=1 -C $REPO && \
+    rm $TAG.tar.gz
+}
+
+gh_retrieve_git(){
+    ORG=$1
+    REPO=$2
+    TAG=$3
+
+    echo "Downloading $ORG $REPO $TAG"
+    git clone --branch $TAG --recursive https://www.github.com/$ORG/$REPO
+}
+
+gh_build(){
+    ORG=$1
+    REPO=$2
+    TAG=$3
+
+    BUILD_CMD=$4
+
+    if [ "${GIT}" ]; then
+        gh_retrieve_git $ORG $REPO $TAG || exit -1
+    else
+        gh_retrieve_wget $ORG $REPO $TAG  || exit -1 
+    fi
+
+    cd $REPO && \
+
+    sh -c "$BUILD_CMD" && \
+    cd .. && \
+    
+    rm -rf $REPO    
+}
+
+
 # Dependencies (not fulfilled by Dockerfile)
-git clone https://github.com/akheron/jansson
-cd jansson ; rm -rf ./.git ; autoreconf -i ; ./configure ; make ; sudo make install ; cd ..
+gh_build "akheron" "jansson" "v2.13.1" "autoreconf -i && ./configure && make && sudo make install" && \
```

Comment:
Up to the project owners, but I would rethink the concatenation of all script lines with that final `&& \`; any command that breaks that "pipeline" / line we may forget to end with the `&& \` sequence will mean that any error generated from that line or the previous ones will be ignored, so very prone to human errors. I think that, if you just skip that final `&& \`, you already have the behavior you want: any `gh_build` call that returns a non 0 exit code will mean that the script will stop executing and return that exit code thanks to the `-e` flag, while in the future, adding lines will not require to remember this trick

Reply:
agreed with @dcaba. I think previous pull requests were addressing this, so this would be a regression